### ISSUE TITLE
The problem with launching applications

### ISSUE THREAD

**Issue by Tarisper** [ahkohd/switch-desktop#64]
Hello! I like your application, but it has two problems:
1. If you run programs using the Alt + Nim hotkeys, they will not start, but a message is displayed that they are already running. Error output "It seems "app name" is yet to be opend. Click to launch app". But the application is not really running.
2. I can not run battle.net app from Blizzard. Error output "Required application data could not found".

OS: Windows 10 1909.

**Comment by ahkohd** [comment]
Hi, @Tarisper thanks for reporting this. Can I have a screenshot the apps on your dock? and if you can list em' that will help too.

**Comment by Tarisper** [comment]
Hi! It's Battle.net error: https://imgur.com/a/VQdBDBk
Error launching other applications does not recur today.

**Comment by Tarisper** [comment]
App:
Battle.net Launcher.exe (https://www.blizzard.com)
steam.exe (steamcommunity.com/)
MS Word
MS Excel
CCleaner
calc.exe
Notepad++
FastStone Image Viewer.

**Comment by ahkohd** [comment]
@Tarisper If I get you right, you mean only Battle.net does not work with Switch and you mean it's probably a battle net issue.

**Comment by Tarisper** [comment]
No. I managed to reproduce the first problem https://imgur.com/a/2aTEa3t
I open the program using the Alt key. I press Alt+2 or other key combinations and the program does not start. When I click on the program icon in Switch, it starts.
